src/views/token-swap/components/selected-quote/selected-quote.view.jsx

```diff
@@ -22,22 +22,6 @@ function SelectedQuote({
 }) {
   const classes = useSelectedQuoteStyles();
 
-  function getRewardAmountInTokens() {
```

Comment:
this is from token swap, so it should stay. It's' a different kind of reward

Reply:
Thx! Reverted!